tests/cli/data/test_cif.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=unused-argument,too-many-arguments
+"""Tests for the `aiida-codtools data cif` CLI command."""
+from __future__ import absolute_import
+
+from uuid import uuid4 as UUID
+
+from aiida import orm
+from aiida_codtools.cli.data.cif import launch_cif_import
+
+
+def test_cif_import(fixture_database, run_cli_command):
+    """Test the `aiida-codtools data cif import` CLI command."""
+    max_entries = 10
+
+    # Default call
+    group = orm.Group(UUID()).store()
+    run_cli_command(launch_cif_import, ['-G', group.pk, '-M', max_entries])
+    assert group.count() == max_entries
+
```

Comment:
should there be a test for long arguments for --group and --max-entries? I do not understand the difference with the -n and -o cases

Reply:
The group flag is determined in `aiida-core` and I just pass it in because it is a required flag. To be honest, I am not even convinced yet that it makes sense to test both flags. The only thing it really does is that the test will break if we change the flag names, but then we just adapt the test. So I think I will just remove the double flag checking